src/drivers/netlify-blobs.ts

```diff
@@ -5,40 +5,42 @@ import type {
   BlobResponseType,
   SetOptions,
   ListOptions,
+  GetStoreOptions,
+  GetDeployStoreOptions,
 } from "@netlify/blobs";
 import { fetch } from "ofetch";
 
 const DRIVER_NAME = "netlify-blobs";
 
 type GetOptions = { type?: BlobResponseType };
 
-export interface NetlifyBaseStoreOptions {
-  /** The name of the store to use. It is created if needed. This is required except for deploy-scoped stores. */
-  name?: string;
+export interface ExtraOptions {
   /** If set to `true`, the store is scoped to the deploy. This means that it is only available from that deploy, and will be deleted or rolled-back alongside it. */
   deployScoped?: boolean;
-  /** Required during builds, where it is available as `constants.SITE_ID`. At runtime this is set automatically. */
-  siteID?: string;
-  /** Required during builds, where it is available as `constants.NETLIFY_API_TOKEN`. At runtime this is set automatically. */
-  token?: string;
-  /** Used for advanced use cases and unit tests */
-  apiURL?: string;
-  /** Used for advanced use cases and unit tests */
-  edgeURL?: string;
```

Comment:
Removal of this, i think breaks types for v7 users (in peer deps, we still allow this)